### PR TITLE
Blog: 12 days of ZAPmas

### PR DESCRIPTION
**Pull request by psiinon**
Ready for review, but not to be published until after Secure Ideas publish theirs, so we can link to it.

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by psiinon** [comment]
Screenshot 2022-12-23 at 16-45-26 The Twelve Days of ZAPmas

File: site/content/blog/2022-12-24-12-days-of-zapmas/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: "The Twelve Days of ZAPmas"
```
Comment (kingthorin) [inline site/content/blog/2022-12-24-12-days-of-zapmas/index.md:2]: > Re: The Twelve Days of ZAPmas"

File: site/content/blog/2022-12-24-12-days-of-zapmas/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+summary: >
+  A reply to an excellent blog series from [Secure Ideas](https://www.secureideas.com/): Twelve Days of ZAPmas - ZAP impressions from a Burp user.
```
Comment (kingthorin) [inline site/content/blog/2022-12-24-12-days-of-zapmas/index.md:4]: Should we stick a 2022 in there to distinguish from any that might occur int he future?

**Review by kingthorin — COMMENTED** [review]
LGTM, just two minor items.

File: site/content/blog/2022-12-24-12-days-of-zapmas/index.md (hunk 2 above)
Comment (psiinon) [reply site/content/blog/2022-12-24-12-days-of-zapmas/index.md:4]: We could add that in the future if new ZAPmas blog posts are made which coulod be confusing?

File: site/content/blog/2022-12-24-12-days-of-zapmas/index.md (hunk 1 above)
Comment (psiinon) [reply site/content/blog/2022-12-24-12-days-of-zapmas/index.md:2]: I'm in 2 minds about this one :)

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

File: site/content/blog/2022-12-24-12-days-of-zapmas/index.md (hunk 2 above)
Comment (kingthorin) [reply site/content/blog/2022-12-24-12-days-of-zapmas/index.md:4]: Sure

**Review by kingthorin — APPROVED** [review]

File: site/content/blog/2022-12-24-12-days-of-zapmas/index.md (hunk 1 above)
Comment (kingthorin) [reply site/content/blog/2022-12-24-12-days-of-zapmas/index.md:2]: 🤷‍♂️ not worth blocking things

**Comment by psiinon** [comment]
Updated to point to https://www.secureideas.com/blog/zapmas-feedback 😁

**Review by kingthorin — APPROVED** [review]

**Pull request merged**